### PR TITLE
Feat/hub feedback

### PR DESCRIPTION
**Pull request by marie-j**
* Add feedback banner for manager hub 

ref: MANAGER-4339

### CODE REVIEW

File: packages/manager/modules/pci/.eslintrc.json [hunk 1]
```diff
@@ -1,7 +1,5 @@
 {
   "globals": {
-    "__FEEDBACK_URL_EN__": true,
```
Comment (antleblanc) [inline packages/manager/modules/pci/.eslintrc.json]: Good catch!

File: packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "manager_hub_dashboard_welcome": "Bienvenue {{ name }} !",
+  "manager_hub_dashboard_feedback": "Nouveau dashboard : n'hésitez pas à nous <a href=\"{{ url }}\" target=\"_blank\">partager vos feedbacks</a>",
```
Comment (antleblanc) [inline packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json]:
```suggestion
  "manager_hub_dashboard_feedback": "Nouveau dashboard : n'hésitez pas à nous <a href=\"{{ url }}\" target=\"_blank\" rel=\"noopener\">partager vos feedbacks</a>",
```

Comment (FredericEspiau) [reply packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json]: Important as it's an actual external link

File: packages/manager/apps/hub/src/index.html [hunk 3]
```diff
@@ -33,7 +37,15 @@
         >
         </ovh-manager-navbar>
         <div class="d-flex">
-            <div data-ui-view class="container-fluid"></div>
+            <div>
```
Comment (FredericEspiau) [inline packages/manager/apps/hub/src/index.html]:
```suggestion
```

File: packages/manager/apps/hub/src/index.js [hunk 4]
```diff
@@ -37,6 +38,7 @@ angular
     /* @ngInject */ ($locationProvider) => $locationProvider.hashPrefix(''),
   )
   .config(routing)
+  .controller('HubController', controller)
```
Comment (FredericEspiau) [inline packages/manager/apps/hub/src/index.js]:
```suggestion
  .controller('hubController', controller)
```

File: packages/manager/apps/hub/webpack.config.js [hunk 5]
```diff
@@ -49,6 +49,12 @@ module.exports = (env = {}) => {
         __NG_APP_INJECTIONS__: process.env.NG_APP_INJECTIONS
           ? `'${process.env.NG_APP_INJECTIONS}'`
           : 'null',
+        __FEEDBACK_URL_EN__: process.env.FEEDBACK_URL_EN
```
Comment (FredericEspiau) [inline packages/manager/apps/hub/webpack.config.js]: I don't get what you are doing, the URL is injected at build time ? Why not hardcoding or APIing it ?

Comment (marie-j) [reply packages/manager/apps/hub/webpack.config.js]: @FredericEspiau we talked about it with @antleblanc this will be changed

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#9991.0 ✘  
* 
  * Build ✘

**Comment by marie-j** [comment]
Updated and fixed conflicts

**Comment by marie-j** [comment]
Fixed conflicts

**Pull request merged**